CountryCode/Numeric.ts

```diff
@@ -17,15 +18,9 @@ export namespace Numeric {
 		740, 744, 748, 752, 756, 760, 762, 764, 768, 772, 776, 780, 784, 788, 792, 795, 796, 798, 800, 804, 807, 818, 826,
 		831, 832, 833, 834, 840, 850, 854, 858, 860, 862, 876, 882, 887, 894, 926,
 	] as const
-	export function is(value: any | Numeric): value is Numeric {
-		return (
-			typeof value == "number" &&
-			value >= 0 &&
-			value <= 999 &&
-			Number.isInteger(value) &&
-			types.includes(value as Numeric)
-		)
-	}
+	export const type = isly.named("isoly.CountryCode.Numeric", isly.number<Numeric>(values as any as number[]))
```

Comment:
if we want to remove the casting we can change
```ts
export const type = isly.named("isoly.CountryCode.Numeric", isly.number<Numeric>(values as any as number[]))
```
to
```ts
export const type = isly.named<Numeric>("isoly.CountryCode.Numeric", isly.number<Numeric>([...values]))
```
since the complaint is that it is a readonly array

Reply:
No, casting does not exist in runtime but creating a full copy of the array is actual work with high impact on the von Neuman bottleneck.

Reply:
But I got away with not having the cast at all as `isly` nowadays takes a readonly array instead.